cmd/mesh/upgrade.go

```diff
@@ -39,6 +39,14 @@ const (
 	// The maximum number of attempts that the command will check for the upgrade completion,
 	// which means only the target version exist and the old version pods have been terminated.
 	upgradeWaitCheckVerMaxAttempts = 60
+
+	// This message provide the guide of how to upgrade Istio data plane
+	upgradeSidecarMessage = "To upgrade the Istio data plane, you will need to re-inject it.\n" +
+		"If you’re using automatic sidecar injection, you can upgrade the sidecar by doing a rolling" +
+		" update for all the pods:\n" +
+		"    kubectl rollout restart deployment --namespace $NAMESPACE_WITH_AUTO_INJECTION\n" +
```

Comment:
$NAMESPACE_WITH_AUTO_INJECTION is a bit confusing - better to represent this as <namespace with auto injection>. Same for $ORIGINAL_DEPLOYMENT_YAML (e.g. <customization YAML file you used to install Istio>